src/errorline.jl

```diff
@@ -141,17 +146,19 @@ end
     for g = axes(y,3) # Iterate through 3rd dimension
         # Compute center and distribution for each value of x
         y_central, y_error = compute_error(y[:,:,g], centertype, errortype, percentiles)
+
+        # Check secondarycolor validity
+        temp = 0;
```

Comment:
Where is that varaible been used?

Reply:
oh whoops, left that in during debugging, 2 seconds